testhelper/docker/resource/internal/ports.go

```diff
@@ -4,7 +4,7 @@ import (
 	"github.com/ory/dockertest/v3/docker"
 )
 
-const BindHostIP = "127.0.0.1"
+const BindHostIP = "0.0.0.0"
```

Comment:
`0.0.0.0` would be corresponding to all the interfaces(including IPV6), we can introduce a new ip for  `172.17.0.1 `

Reply:
`host.docker.internal` doesn't always resolve to the same ip...

Reply:
would something like this help? https://github.com/rudderlabs/rudder-go-kit/pull/615/files

Reply:
In case of Docker Desktop for Mac, this causes issues :( 
```
API error (500): driver failed programming external connectivity on endpoint thirsty_napier (78e1f6fa6a0132e1f00cbadbb20ef86b89d7cdc55ddd6828c0eb7bbc9516d67a): failed to bind port 127.0.0.1:55053/tcp: listen tcp4 127.0.0.1:55053: bind: address already in use
```

Reply:
@mihir20 the only way I can think right now is in #616